src/libraries/Microsoft.Extensions.Primitives/tests/StringSegmentTest.cs

```diff
@@ -453,20 +444,13 @@ public void StringSegment_StartsWith_String_InvalidComparisonType_Throws()
             Assert.Throws<ArgumentOutOfRangeException>("comparisonType", () => segment.StartsWith(string.Empty, (StringComparison)6));
         }
 
-        public static TheoryData<string, StringComparison, bool> EqualsStringData
+        // candidate / comparer / expected result
+        public static TheoryData<string, StringComparison, bool> EqualsStringData =>new()
         {
-            get
-            {
-                // candidate / comparer / expected result
-                return new TheoryData<string, StringComparison, bool>()
-                {
-                    { "eLLo", StringComparison.OrdinalIgnoreCase, true },
-                    { "eLLo", StringComparison.Ordinal, false },
-                    { null, StringComparison.OrdinalIgnoreCase, false },
```

Comment:
Why is `{ null, StringComparison.OrdinalIgnoreCase, false },` being removed?

Reply:
That's strange. I looked through the commits, and I've never removed it.
Added it back.